### PR TITLE
hotfix add airflow version

### PR DESCRIPTION
**Pull request by andriisoldatenko**
## Description

Add `airflowVersion` to helm values during createDeployment #2094

## 🎟 Issue(s)

Resolves https://github.com/astronomer/issues/issues/2094

## 🧪 Functional Testing

practically LGTM:

```
quotas:
    pods: 16
    requests.cpu: 7200m
    requests.memory: 27648Mi
    limits.cpu: 7200m
    limits.memory: 27648Mi
defaultAirflowTag: 1.10.10-5-alpine3.10-onbuild
airflowVersion: 1.10.10
labels:
    platform: deadly-terrier
    workspace: ckh59o04n0161zvxtbk5et381
    revision: '1'
platform:
    release: deadly-terrier
    workspace: ckh59o04n0161zvxtbk5et381
executor: CeleryExecutor

2020-11-05T22:12:35 INFO Commander disabled, skipping call to #createDeployment

```

### CODE REVIEW

File: src/lib/deployments/config/index.js [hunk 1]
```diff
@@ -64,6 +63,7 @@ export async function generateHelmValues(deployment, values = {}) {
     registry(deployment), // Apply the registry connection details.
     elasticsearch(deployment), // Apply the elasticsearch connection details.
     await defaultAirflowTag(deployment), // Apply the default airflow tag we want to deploy.
+    { airflowVersion: get(deployment, "airflowVersion") }, // Apply airflow version
```
Comment (schnie) [inline src/lib/deployments/config/index.js:66]: @andriisoldatenko could keep the pattern and call a function to return this object here?

or maybe combine `defaultAirflowTag()` and this new line into a single `airflowVersion()` function that does both? Essentially combine the existing function with this and rename it to be more generic. What do you think?

Comment (andriisoldatenko) [reply src/lib/deployments/config/index.js:66]: @schnie  `airflowValues()`?

Comment (schnie) [reply src/lib/deployments/config/index.js:66]: I'd say just `airflowVersion` since both values we need to set are based on the actual `airflowVersion`.

**Review by schnie — APPROVED** [review]
LGTM

**Comment by andriisoldatenko** [comment]
practically LGTM:
```
quotas:
    pods: 16
    requests.cpu: 7200m
    requests.memory: 27648Mi
    limits.cpu: 7200m
    limits.memory: 27648Mi
defaultAirflowTag: 1.10.12-1-alpine3.10-onbuild
airflowVersion: 1.10.12
labels:

```

**Pull request merged**